### ISSUE TITLE
Features dropdown cannot be blank when modifying a rapid experiment

### ISSUE THREAD

**Issue by pdehaan** [mozilla/experimenter#2934]
https://stage.experimenter.nonprod.dataops.mozgcp.net/experiments/rapid/test-aaaaaaa2/edit/

I was able to create a rapid experiment on stage and click the "Save" button and then review said experiment. Unhappy with my dummy data, I clicked the "Back" button to modify a value, but now when I click the "Save" button, the Features select field is now required (even though it wasn't when I was creating the rapid experiment).

![Mozilla_Experimenter](https://user-images.githubusercontent.com/557895/86059086-ead38900-ba16-11ea-9336-502ecbd88103.png)

**Comment by pdehaan** [comment]
Not sure if this is a blocker. I now cannot Save this experiment or return to the screen that lets me submit for approval. Also uncertain if there are other errors on the firm (required Audience or Trigger?) that are being hidden by this error. I'd have try digging through the source to know if I'd see all errors at once on the form, or if I'd have to solve one before seeing the others.

**Comment by pdehaan** [comment]
Here's the UI when I try and submit a new/blank rapid experiment (note that it only tells/enforces that the "Public Name" and "Hypothesis" are required during creation). It's only after you click Save => Back that "Features" becomes required.

![Mozilla_Experimenter](https://user-images.githubusercontent.com/557895/86062355-60daee80-ba1d-11ea-8f4f-5b86e02072cd.png)

**Comment by jaredlockhart** [comment]
Closing as a duplicate of https://github.com/mozilla/experimenter/issues/2843
